releasenotes/README.md

```diff
@@ -25,12 +25,40 @@ Additional documentation:
 * [Salesforce CLI Plugin Developer Guide](https://github.com/salesforcecli/cli/wiki/Quick-Introduction-to-Developing-sf-Plugins)
 * [Salesforce CLI Setup Guide](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_intro.htm)
 
-## 2.72.21 (Jan 15, 2025) [stable-rc]
+## 2.73.6 (Jan 22, 2025) [stable-rc]
 
 These changes are in the Salesforce CLI release candidate. We plan to include these changes in next week's official release. This list isn't final and is subject to change.
 
 ------------
 
+* NEW: Ensure that your code adheres to best practices with these Code Analyzer v5 (Beta) commands in the just-in-time (JIT) `code-analyzer` plugin:
+
+    * `code-analyzer config` :  Display the current state of configuration for Code Analyzer.
+    * `code-analyzer rules` : List the rules that are available to analyze your code.
+    * `code-analyzer run`  :  Analyze your code with a selection of rules to ensure good coding practices.
+
+   The `code-analyzer` plugin isn't included in the core Salesforce CLI; instead, it's now installed the first time you run one of its commands. 
+
+   The Code Analyzer v4 commands, such as `scanner run`, are also JIT and continue to work the same as before. Because the v4 and v5 commands are in separate CLI topics and plugins, they don't interfere with each other.
```

Comment:
Do you want to mention that the `scanner` commands will be removed (from JIT) in the future? I don't think we have an exact date yet.